packages/by-name/contrast/package.nix

```diff
@@ -51,7 +51,12 @@ let
     let
       runtimeHandler =
         platform: hashFile:
-        "contrast-cc-${platform}-${builtins.substring 0 8 (builtins.readFile hashFile)}";
+        let
+          runtimeHash = builtins.readFile hashFile;
+          inherit (contrast) version;
+          runtimeVersionHash = builtins.hashString "sha256" "${version}${runtimeHash}";
```

Comment:
I don't really like doing this here, pretty hard to understand the hash if it is extended here and there. Can we add the version file as layer into the node-installer-image and include it in the hash creation in https://github.com/edgelesssys/contrast/blob/main/packages/by-name/kata/contrast-node-installer-image/package.nix#L261-L264 ?

Reply:
Added a layer containing the version file.